os-server/src/main/java/app/openschool/coursemanagement/entity/Status.java

```diff
@@ -0,0 +1,82 @@
+package app.openschool.coursemanagement.entity;
+
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "status")
+public class Status {
```

Comment:
I guess we should store statuses of modules, courses and module items into a separate tables